R/R.sublime-syntax

```diff
@@ -29,8 +30,15 @@ contexts:
     - include: function-calls
     - include: general-variables
 
+  codesection:
+    - match: ^\s*(\#+)\s*(.+?)\s*[-=#]{4,}$\n?
```

Comment:
I think this needs to be something like:

```
^\s*(\#+)\s*(.+?)\s*(?:-{4,}|={4,}|#{4,})[ \t]*$\n?
```

The current regex would allow:

```
# Code section -=#=-
```